scripts/build-all.js

```diff
@@ -5,6 +5,7 @@ const rxjs = require('rxjs');
 const operators = require('rxjs/operators');
 const cliProgress = require('cli-progress');
 const chalk = require('chalk');
+const { exit } = require('process');
```

Comment:
is deze nodig ? je lijkt hier niets mee te doen?

Reply:
Goed gezien. Verwijderd!